packages/util-arn-parser/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "@aws-sdk/util-arn-parser",
+  "version": "1.0.0-gamma.0",
+  "description": "A parser to Amazon Resource Names",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "scripts": {
+    "prepublishOnly": "yarn build:cjs && yarn build:es",
+    "pretest": "yarn build:cjs",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
```

Added build command like it's done in https://github.com/aws/aws-sdk-js-v3/pull/1407
```suggestion
    "build:es": "tsc -p tsconfig.es.json",
    "build": "yarn build:es && yarn build:cjs"
```